wallet/walletrpc.c

```diff
@@ -177,29 +177,54 @@ static const struct json_command newaddr_command = {
 };
 AUTODATA(json_command, &newaddr_command);
 
-static struct command_result *json_listaddrs(struct command *cmd,
+static void json_add_addr(struct json_stream *response,
+				 const u64 keyidx,
+				 const struct pubkey *pubkey,
+				 const char *out_p2wpkh,
+				 const u8 *redeemscript_p2wpkh,
+				 const char *out_p2tr)
+{
+	json_object_start(response, NULL);
+	json_add_u64(response, "keyidx", keyidx);
+	json_add_pubkey(response, "pubkey", pubkey);
+	json_add_string(response, "bech32", out_p2wpkh);
+	json_add_hex_talarr(response, "bech32_redeemscript", redeemscript_p2wpkh);
```

Comment:
Why is `pubkey` and `bech32_redeemscript` being returned? I don't see a use for either

Reply:
`bech32_redeemscript` is being used in our test but it made sense to not expose it to the user.  `pubkey` & `bech32_redeemscript` are now returned only when `--developer` mode is enabled.